app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -186,6 +186,10 @@ def destroy_vm(uuid)
       true
     end
 
+    def supports_vms_pagination?
+      true
```

Comment:
oic, i suggest we release a 0.0.x version instead atm, and we should make
sure katello can run with the latest rest-client.